src/web/components/ApiKeyManagement/KeyItem.tsx

```diff
@@ -42,6 +50,13 @@ function KeyItem({ apiKey: apiKeyInitial, onEdit, onDisable, availableRoles }: K
       {availableRoles.length > 0 && (
         <td className='action'>
           <div className='action-cell'>
+            {keyNeedsRotating && (
+              <FontAwesomeIcon
+                icon='triangle-exclamation'
+                title='We recommend you rotate your API key every year.'
+                className='warning-button'
```

Comment:
nit: perhaps `warning-icon` rather than `warning-button` given it's not a button

Reply:
Chagned - at first I thought it would end up being a button if/when we rotate keys directly from the portal but now that I think about it, the button would probably not be the warning icon